DashWallet/Sources/Models/CrowdNode/CrowdNode.swift

```diff
@@ -127,7 +127,6 @@ public final class CrowdNode {
     init() {
         masternodeAPY = DWEnvironment.sharedInstance().apy.doubleValue
         crowdnodeAPY = masternodeAPY * 0.85
-        accountAddress = prefs.crowdNodeAccountAddress ?? "" // Restore from user default
```

Comment:
This seems to be causing some problems. When we change the network, sometimes the saved address sticks for some reason and we get the wrong state for CrowdNode.

I figure we can get the saved address from the `FullCrowdNodeSignUpTxSet` directly and keep the normal restoring logic in here to avoid issues.